demofile-parsing/index.js

```diff
@@ -0,0 +1,63 @@
+const inquirer = require('inquirer');
+
+console.log(`Welcome to the Highlight Generator Tool`);
+console.log();
+var testURL = "https://www.twitch.tv/videos/449439967"; // Change according to Testing needs
+var testFilePath = "C:/Users/eldri/Documents/GitHub/Local_Files/heroic-vs-ence-mirage.dem"; // Change according to Testing needs
+var testStartTime = 2000; // Change according to Testing needs
+
+const TESTING = false; //Turn this on to skip prompts when testing
```

Comment:
I like the style of these DEBUG/TESTING bool switches - lets maybe have them ingest a debug arg in the future i.e. `node index.js debug=ON`